ui/package.json

```diff
@@ -67,6 +67,7 @@
     "prop-types": "^15.6.2",
     "qs": "^6.6.0",
     "react": "^16.8.2",
+    "ramda": "^0.26.1",
```

Comment:
We used ramda in many places in the UI. Not sure why it wasn't specified in package.json.

Reply:
Looks like material-ui-treeview depends on it.